recipes/package.rb

```diff
@@ -1,6 +1,6 @@
-if(node[:ruby_installer][:package_name])
-  if(node[:ruby_installer][:package_name] == true)
-    case node.platform_family
+if node['ruby_installer']['package_name']
```

Comment:
Just to make sure I'm understanding the logic here, if this attribute is `nil` or `false`, we won't install a package (why are you running this recipe?) If it's `true`, we use sensible defaults to decide when package to install. If it's some other value, we use that as the name of the package to install. 

Does that match your understanding?